### DISCUSSION TITLE
Any project that has used fastapi-users on github?

### DISCUSSION THREAD

**Discussion by johnsxiongs** [fastapi-users/fastapi-users#427] [General]
Any project that has used fastapi-users on github?
I just want to clone it and learn how it work!

**Comment by frankie567** [comment]
Here are some projects I've found that could help:

* https://github.com/ankushjain2001/fastapi-react-mongodb
* https://github.com/OlegBugaichuk/edu_platform
* https://github.com/skapralov/english_coach_backend
* https://github.com/ankushjain2001/fastapi-react-mongodb

Cheers!
